src/test/java/com/wikia/webdriver/common/skin/SkinHelper.java

```diff
@@ -14,24 +15,37 @@
   @FindBy(css = "body.ember-application")
   private WebElement mercuryClassInBody;
 
-  @FindBy(css = "body.wkMobile")
-  private WebElement wikiaMobileClassInBody;
+  @FindBy(css = "body.ember-application.mobile-wiki")
+  private WebElement mobileWikiClassInBody;
 
   public SkinHelper(WebDriver driver) {
     super();
   }
 
   public boolean isSkin(Skin skin) {
+    boolean isExpectedSkin;
+
     switch (skin) {
       case OASIS:
-        return wait.forElementInViewPort(oasisClassInBody);
+        isExpectedSkin = wait.forElementInViewPort(oasisClassInBody);
+        break;
       case MERCURY:
-        return wait.forElementInViewPort(mercuryClassInBody);
-      case WIKIAMOBILE:
-        return wait.forElementInViewPort(wikiaMobileClassInBody);
+        isExpectedSkin = wait.forElementInViewPort(mercuryClassInBody);
+        break;
+      case MOBILE_WIKI:
+        isExpectedSkin = wait.forElementInViewPort(mobileWikiClassInBody);
+        break;
       default:
-        return false;
+        isExpectedSkin = false;
+    }
+
+    if (isExpectedSkin) {
+      PageObjectLogging.logInfo("Expected skin is loaded: " + skin.toString());
+    } else {
+      PageObjectLogging.logWarning("isSkin", "Expected skin is not loaded: " + skin.toString());
```

Comment:
Most of the times we use it like this:
```
Assertion.assertTrue(new SkinHelper(driver).isSkin(Skin.MOBILE_WIKI));
```
Not sure if we should do the assertion in this method which can be (and is) used without it.